tests/engine/test_filter_evttype_resolver.cpp

```diff
@@ -0,0 +1,237 @@
+/*
```

Comment:
So libsinsp/test/evttype_filter.ut.cpp will be removed after this is merged?

Reply:
Exactly, we have one in WIP to be merged after this one: https://github.com/falcosecurity/libs/pull/280